libs/analytics/WordPressAnalytics/src/main/java/org/wordpress/android/analytics/AnalyticsTrackerNosara.java

```diff
@@ -273,13 +273,13 @@ public void track(AnalyticsTracker.Stat stat, Map<String, ?> properties) {
                 predefinedEventProperties.put("menu_item", "site_settings");
                 break;
             case OPENED_ACCOUNT_SETTINGS:
-                eventName = "me_opened_account_settings";
+                eventName = "me_account_settings_opened";
```

Comment:
Can we drop the `me_` prefix on both of these events? There has been some history already on the iOS side with `account_settings_opened` so I don't want to affect any funnels already using that. Although I can be compelled to add `me_` to iOS if @daniloercoli feels strongly about it.